spec/lib/fake_data/complaints_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe FakeData::Complaints do
+  describe "jsonapi_wrapper" do
+    it "should return a object with JSON-API top level JSON API" do
+      wrapper = FakeData::Complaints.jsonapi_wrapper
+
+      expect(wrapper[:meta]).to be_a(Hash)
+      expect(wrapper[:data]).to be_a(Array)
+      expect(wrapper[:links]).to be_a(Hash)
+    end
+  end
+
+  describe "complaint_data" do
+    it "should return a single complaint object" do
+      complaint = FakeData::Complaints.complaint_data
+
+      expect(complaint[:id]).to_not be_nil
```

Comment:
[suggestion] do we want to expect that this is an integer?

Reply:
cool, let's expect it to be the thing it is!

Reply:
Okay!  I was using the type from HSES's issues table, but I think it makes sense to be a string for flexibility.

Reply:
I will update the Open API spec for the identifiers, also

Reply:
This test is still missing the verification that `[:id]` is a string